src/website/src/pages/ec/components/button/docs/usage.mdx

```diff
@@ -9,33 +9,54 @@ import { Paragraph } from '@ecl/website-components';
   Use a button to trigger and carry out an action on a page.
 </Paragraph>
 
+## Anatomy
+
+Buttons use 3 major attributes:
+
+- Labels
+  - Should be clear and distinct.
+  - Use verbs to describe the action performed when interacting with the button.
+  - Keep it short, no more than 3 words and only capitalise the first word.
+- Colour
```

Comment:
The color is set by the button variant. As far as I know this is not possible/recommanded to change it

Reply:
Still the point is valid, I guess what the author meant is "in the case of ghost button"...